.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,128 @@
+        type: string
+        required: false
+        default: ''
+      branch:
+        description: "Branch name"
+        type: string
+        required: true
+        default: 'cleanup-python-code'
+      draft:
+        description: "Create draft PR"
+        type: boolean
+        required: false
+        default: false
+      force_delete_old_prs:
+        description: "Forcefully delete old PRs"
+        type: boolean
+        required: false
+        default: false
+      script:
+        description: "Script"
+        type: string
+        required: true
+
+jobs:
+
+  repos_list:
+    runs-on: ubuntu-20.04
+
+    outputs:
+      output1: ${{ steps.repos_list.outputs.list }}
+    steps:
+
+    - name: get repos list
+      id: repos_list
+      run: |
+        echo "::set-output name=list::[${{github.event.inputs.repos_list}}]"
+
+  bulk_update:
+    runs-on: ubuntu-20.04
+    needs: [ repos_list ]
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ${{fromJson(needs.repos_list.outputs.output1)}}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+            repository: ${{ github.event.inputs.organization}}/${{ matrix.repos }}
+            ref: 'master'
+
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ github.event.inputs.python_version }}
+
+      - name: install packages
+        if: ${{ github.event.inputs.packages }}
+        run: pip install ${{ github.event.inputs.packages }}
+
+      - name: run script
+        run: ${{ github.event.inputs.script }}
+
+      - name: setup testeng-ci
+        run: |
+          git clone https://github.com/edx/testeng-ci.git
+          cd $GITHUB_WORKSPACE/testeng-ci
+          pip install -r requirements/base.txt
+
+      - name: setup draft flag
+        run: echo "draftflag=$(if ${{ github.event.inputs.draft }}; then echo '--draft'; else echo ''; fi)" >> $GITHUB_ENV
+
+      - name: setup force delete flag
+        run: echo "force_delete_old_prs_flag=$(if ${{ github.event.inputs.force_delete_old_prs }}; then echo '--force-delete-old-prs'; else echo '--no-force-delete-old-prs'; fi)" >> $GITHUB_ENV
+
+      - name: ignore testeng-ci
+        run: echo "testeng-ci" >> .git/info/exclude
+
+      - name: create pull request
+        id: createpullrequest
+        env:
+          GITHUB_TOKEN: ${{ secrets.requirements_bot_github_token }}
+          GITHUB_USER_EMAIL: ${{ secrets.requirements_bot_github_email }}
+        run: |
+          cd $GITHUB_WORKSPACE/testeng-ci
+          python -m jenkins.pull_request_creator --repo-root=$GITHUB_WORKSPACE \
+          --target-branch="master" --base-branch-name="${{ github.event.inputs.branch }}" \
+          --commit-message="${{ github.event.inputs.commit_message }}" \
+          --pr-title="${{ github.event.inputs.commit_message }}"  \
+          --pr-body="${{ github.event.inputs.pr_body }}" --team-reviewers="arbi-bom" \
```

Comment:
We should opt for the `user-reviewer` choice even if tagging fails for some users. We can add a warning message in case the job fails to tag the mentioned user. 
Tagging our team should be avoided so we don't get pinged when some other team or user uses this action for their tasks.